docx2html/core.py

```
@@ -839,13 +865,39 @@ def get_list_data(li_nodes, meta_data):
     # Store the first list created (the root list) for the return value.
     root_ol = None
     visited_nodes = []
+    texts = []
+
+    # Helper function
+    def _build_li(texts):
+        data = '<br/>'.join(t for t in texts if t is not None)
+        return etree.XML('<li>%s</li>' % data), []
+
     for li_node in li_nodes:
         w_namespace = get_namespace(li_node, 'w')
+        if not is_li(li_node, meta_data):
+            if li_node.tag == '%stbl' % w_namespace:
+                table_el, el_visited_nodes = get_table_data(li_node, meta_data)
+                texts.append(etree.tostring(table_el))
+            elif li_node.tag == '%sp' % w_namespace:
+                # This is a simple paragraph
+                texts.append(get_p_data(
+                    li_node,
+                    meta_data,
+                ))
+                el_visited_nodes = [li_node]
+            else:
+                # I don't know what this tag is.
```

Doesn't this mean that if we hit something other than a paragraph or table inside a list, we might lose that content? Wouldn't it be better to raise an Exception here rather than potentially drop data?